packages/services/src/services/Translation/Translation.js

```diff
@@ -110,10 +111,11 @@ class TranslationAPI {
    *
    * @param {string} lang Language code
    * @param {string} country Country code
+   * @param {string} endpoint endpoint to fetch data
    * @param {Function} resolve resolves the Promise
    * @param {Function} reject rejects the promise
    */
-  static fetchTranslation(lang, country, resolve, reject) {
+  static fetchTranslation(lang, country, endpoint, resolve, reject) {
```

Comment:
@annawen1 should we also offer this up as an environment variable? Perhaps instead of `_dds_endpoint` we open up `DDS_TRANSLATION_ENDPOINT`.

The other thing we might want to track is the sessionStorage key being used, as we don't want a custom endpoint data to conflict with the default. Maybe we can:
1) Set up DDS_TRANSLATION_ENDPOINT
2) If DDS_TRANSLATION_ENDPOINT is not defined, use _dds_endpoint
3) Also have `endpoint` as an optional attribute for the service
4) logic would be `endpoint attribute` used first if defined, if not then use `DDS_TRANSLATION_ENDPOINT`
5) if `DDS_TRANSLATION_ENDPOINT` does not match the default value, set the `_sessionTranslationKey` to something different (perhaps derived somehow from the custom defined `DDS_TRANSLATION_ENDPOINT`

Reply:
yep I can set the env var up, the session key I'll see what might be the best way to get the key name from the endpoint...